config/prod/test-size-change.yaml

```diff
@@ -0,0 +1,24 @@
+# Provision an EC2 instance
+template_path: templates/managed-ec2-v3.yaml
+stack_name: iwanna-instance
+parameters:
+  # The Sage deparment for this resource
+  Department: "Platform"
+  # The Sage project this resource will be used for
+  Project: "Infrastructure"
+  # The resource owner
+  OwnerEmail: "joe.smith@sagebase.org"
```

Comment:
you need to provide your email otherwise you won't get the ip address for the provisioned EC2.